pages/resources/quests.mdx

```diff
@@ -0,0 +1,518 @@
+| last_stream_heartbeat_ack | ISO8601 timestamp | Timestamp of the latest stream heartbeat request                                                                                |
+| stream_progress_seconds   | ISO8601 timestamp | The total amount of seconds the user has been streaming the game for since the quest started and before the quest was completed |
+| dismissed_quest_content   | ISO8601 timestamp | Whenever the user has dismissed the quest                                                                                       |
+| progress                  | object            | Unused                                                                                                                          |
+
+##### Example User Status
+
+```json
+{
+  "user_id": "222069018507345921",
+  "quest_id": "alien-quest",
+  "enrolled_at": "2077-01-01T11:59:59+00:00",
+  "completed_at": "2077-01-01T11:59:59+00:00",
+  "claimed_at": "2077-01-01T11:59:59+00:00",
+  "claimed_tier": null,
+  "last_stream_heartbeat_at": "2077-01-01T11:59:59+00:00",
+  "stream_progress_seconds": 1800,
+  "dismissed_quest_content": 0,
+  "progress": {}
+}
+```
+
+### Quest Assets Object
+
+Represents an object holding image file names, the assets can be accessed via the [quest asset endpoint](#quest-assets-endpoint)
+
+###### Quest Assets Structure
+
+| Field          | Type   | Description                                                                                                                                         |
+| -------------- | ------ | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| reward_tile    | string | An image showing a preview of the reward                                                                                                            |
+| hero           | string | The quest's [hero image](https://en.wiktionary.org/wiki/hero_image), aka, fancy background                                                          |
+| quest_bar_hero | string | The [hero image](https://en.wiktionary.org/wiki/hero_image) used in the quest popup that appears when launching the game before accepting the quest |
+| game_tile      | string | The game's icon                                                                                                                                     |
+| logotype       | string | The game's [logotype](https://en.wikipedia.org/wiki/Logo)                                                                                           |
+
+## Example Quest Assets Object
+
+```json
+{
+  "reward_tile": "reward.png",
+  "hero": "hero.png",
+  "quest_bar_hero": "quests_bar_hero.png",
+  "game_tile": "game_tile.png",
+  "logotype": "game_logotype.png"
+}
+```
+
+### Quest Reward Object
```

Comment:
```suggestion
### Quest Reward Code Object
```
This should probably renamed to reward code object so the other one can be called reward object